editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/impl/simple/AbstractCBRTemplate.java

```diff
@@ -54,5 +54,10 @@ public CBRUnzipTemplateCreator(String suffix) {
 		}
 
 	}
+	
+	@Override
+	public boolean isDefault(EnvironmentData environment, CamelDSLType dslType) {
+		return isCompatible(environment) && CamelDSLType.BLUEPRINT.equals(dslType);
+	}
 
 }
```

Comment:
github wants a new line at the end

Reply:
GitHub wants nothing, GitHub is pointing out that there is no new-line at the end of the file. And they recommend to put one as diff support for next PR is simplified. This is not modified code but I can modify it, it will just increase the number of modifications.

Reply:
If it helps for the future diffs then I opt for adding the new line.